src/styles/App.js

```diff
@@ -0,0 +1,57 @@
+import background_image from '../assets/images/background.jpg';
+import {
+    themeColor,
+    silverColor,
+    yaQingColor,
+}from '../constants';
+
+const styles = {
+    container: {
+        //background
+        width: '100vw',
+        height: '100vh',
+        backgroundImage: `url(${background_image})`,
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+        //items in it
+        display: 'flex',
+        justifyContent: 'center',
+    },
+    panel: {
+        marginTop: 60,
+        width: 500,
+        height: 600,
+        backgroundColor: '#ffffff',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    logo: {
+        color: themeColor,
+        fontSize: 50,
+        fontFamily: 'RussoOne',
+        marginTop: 46,
+        marginBottom: 32,
+    },
+    footer_container: {
+        
+    },
+    footer: {
+        width: 500,
+        height: 118,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: silverColor,
+    },
+    footer_text: {
+        fontSize: 25,
```

Comment:
需要显式生命字体，某则采用`index.css`中的默认设置
```suggestion
        fontSize: 25,
        fontFamily: 'Roboto-Regular'
```